lib/libyui_client/widgets/tree.rb

```diff
@@ -0,0 +1,160 @@
+# frozen_string_literal: true
+
+module LibyuiClient
+  module Widgets
+    # Class representing a tree in UI. It can be YTree.
+    class Tree < Widgets::Base
+      # Returns the list of items available to select from tree. Each item
+      # is represented by its path from root node to leaf in the tree,
+      # in other words, a list of nodes separated by special simbol '|'
+      # @return [Array] array of Strings.
+      # @example Get items from tree with id "test_id"
+      #  {
+      #    "class": "YTree",
+      #    "debug_label": "node_0",
+      #    "hstretch": true,
+      #    "hweight": 30,
+      #    "icon_base_path": "",
+      #    "id": "test_id",
+      #    "items": [
+      #      {
+      #        "children": [
+      #          {
+      #            "icon_name": "icon",
+      #            "label": "node1_1"
+      #          },
+      #          {
+      #            "children": [
+      #              {
+      #                "label": "node1_2_1"
+      #              },
+      #              {
+      #                "label": "node1_2_2",
+      #                "selected": true
+      #              }
+      #            ],
+      #            "icon_name": "icon",
+      #            "label": "node1_2"
+      #          }
+      #        ],
+      #        "icon_name": "icon",
+      #        "label": "node1",
+      #      },
+      #      {
+      #        "icon_name": "icon",
+      #        "label": "node2"
+      #      }
+      #    ],
+      #    "items_count": 2,
+      #    "label": "node_0",
+      #    "notify": true,
+      #    "vstretch": true
+      #  }
+      # @example
+      #   app.tree(id: 'test_id').items
+      #   # node1
+      #   # node1|node1_1
+      #   # node1|node1_2
+      #   # node1|node1_2|node1_2_1
+      #   # node1|node1_2|node1_2_2
+      #   # node2
+      def items
+        traversed_nodes = []
+        traverse_tree(property(:items), traversed_nodes: traversed_nodes)
+        traversed_nodes
+      end
+
+      # Sends action to select an item from the tree in UI.
+      # @param item [String] value to select from items.
+      # @example Select tree item 'test_item' from tree with id 'test_id'
+      #   app.tree(id: 'test_id', value: test_item).select
```

Comment:
As I understand from the code below, the click should be made with different syntax: `app.tree(id: 'test_id').select('test_item')`. Could you please update the documentation.

Reply:
fixed